loleaflet/Makefile.am

```diff
@@ -28,9 +28,9 @@ $(L10N_IOS_ALL_JS) : $(wildcard $(srcdir)/po/ui-*.po) $(shell find $(srcdir)/l10
 endif
 
 SRC_TS = $(wildcard src/layer/tile/*.ts)
-SRC_TS_JS = $(patsubst src/layer/tile/%.ts,$(DIST_FOLDER)/src/layer/tile/%.js,$(SRC_TS))
```

Comment:
Shouldn't the changes to Makefile.am be part of the first commit where the .ts file was added ? ("git rebase -i" can fix this)